src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -4,5 +4,18 @@
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
+    private static final int ARRAY_SIZE = 6;
```

Comment:
Avoid using static methods or variables unless they are constants or utilities. In this case, ARRAY_SIZE is a constant, which is good practice, but ensure that any other static usage is justified.